src/jdk.httpserver/share/classes/module-info.java

```diff
@@ -37,6 +37,58 @@
+ * <p>
+ * <b id="httpserverprops">System properties used by the HTTP server API</b>
+ * <p>
+ * The following is a list of JDK specific system properties used by the HTTP
+ * server implementation.
+ * <ul>
+ * <li><p><b>{@systemProperty sun.net.httpserver.idleInterval}</b> (default: 30 sec)<br>
+ * Maximum duration in seconds which an idle connection is kept open. This timer
+ * has an implementation specific granularity that may mean that idle connections are
+ * closed later than the specified interval.
+ * </li>
+ * <li><p><b>{@systemProperty jdk.httpserver.maxConnections}</b> (default: -1)<br>
+ * The maximum number of open connections at a time. This includes active and idle connections.
+ * If zero or negative, then no limit is enforced.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxIdleConnections}</b> (default: 200)<br>
+ * The maximum number of idle connections at a time.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.drainAmount}</b> (default: 64K)<br>
+ * The maximum number of bytes that will be automatically read and discarded from a
+ * request body that has not been completely consumed by its
+ * {@link com.sun.net.httpserver.HttpHandler HttpHandler}. If the number of remaining
+ * unread bytes are less than this limit then the connection will be put in the idle connection
+ * cache. If not, then it will be closed.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxReqHeaders}</b> (default: 200)<br>
+ * The maxiumum number of header fields accepted in a request. If this limit is exceeded
+ * while the headers are being read, then the connection is terminated and the request ignored.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxReqTime}</b> (default: -1)<br>
+ * The maximum duration in milliseconds of a request. In practice, this is a function of
+ * request size, network speed, and handler processing delays. A value less than or equal
+ * to zero means the duration is not limited. If the limit is
+ * exceeded then the connection is terminated and the handler will receive a
+ * {@link java.io.IOException}. This timer has an implementation specific granularity
+ * that may mean requests are aborted later than the specified interval.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.maxRspTime}</b> (default: -1)<br>
+ * The maximum duration in milliseconds of a response. In practice, this is a function of
+ * response size, network speed, and handler processing delays. A value less than or equal
+ * to zero means the duration is not limited. If the limit is
+ * exceeded then the connection is terminated and the handler will receive a
+ * {@link java.io.IOException}. This timer has an implementation specific granularity
+ * that may mean responses are aborted later than the specified interval.
+ * </li>
+ * <li><p><b>{@systemProperty sun.net.httpserver.debug}</b> (default: false)<br>
+ * Boolean value, which if true, generates debug information on the console.
```

Comment:
Should we instead say that it generates the debug information on `{@link System#out}`, since that's where we direct those logs and the `System.out` can be anything (if the application changes it to something else).

Reply:
This looks like debug and it probably shouldn't be documented - or possibly replace by regular logger calls since the httpserver uses java.util.logging. While is the reason for using `dprint` instead of regular j.u.l as is done elsewhere in the server implementation?

Reply:
Good idea!